plugins/inputs/opcua_listener/subscribe_client.go

```diff
@@ -78,7 +80,21 @@ func assignConfigValuesToRequest(req *ua.MonitoredItemCreateRequest, monParams *
 	return nil
 }
 
+func (sc *SubscribeClientConfig) Validate() error {
+	switch sc.ConnectFailBehavior {
+	case "error", "ignore", "retry":
```

Comment:
Handle the default here and remove the setting in `Init()` (and in the tests)...
```suggestion
	switch sc.ConnectFailBehavior {
	case "":
		sc.ConnectFailBehavior = "error"
	case "error", "ignore", "retry":
```